### PR TITLE
Post transactions support

### PR DESCRIPTION
**Pull request by qnikst**

### CODE REVIEW

File: src/rest/v0/transaction/mod.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+use actix_web::{App, Error as ActixError, HttpMessage, HttpRequest, Responder};
+use bytes::IntoBuf;
+use chain_core::property::Deserialize;
+use chain_impl_mockchain::transaction::SignedTransaction;
+use futures::Future;
+use std::sync::{Arc, Mutex};
+
+pub type Task = Arc<Mutex<TaskMessageBox<TransactionMsg<Mockchain>>>>;
+
+pub fn crate_handler(
```
Comment (mzabaluev) [inline src/rest/v0/transaction/mod.rs]: Why is this named `crate_handler`?

Comment (CodeSandwich) [reply src/rest/v0/transaction/mod.rs]: Because it creates item, which then can be passed to `add_handler`

Comment (mzabaluev) [reply src/rest/v0/transaction/mod.rs]: But what is a "crate", in this context?

Comment (CodeSandwich) [reply src/rest/v0/transaction/mod.rs]: :D I'll fix it

**Pull request merged**